src/events/mention/MentionDeleteEventHandler.ts

```diff
@@ -0,0 +1,27 @@
+import { MessageReaction, User } from 'discord.js';
+import * as log4js from 'log4js';
+import BotConfig from '../../BotConfig';
+import EventHandler from '../EventHandler';
+
+export default class MentionDeleteEventHandler implements EventHandler<'messageReactionAdd'> {
+    public readonly eventName = 'messageReactionAdd';
+
+    private logger = log4js.getLogger( 'MentionDeleteEventHandler' );
+
+    public onEvent = async ( { message }: MessageReaction, user: User ): Promise<void> => {
+        this.logger.info( `User ${ user.tag } is attempting to delete message '${ message.id }'` );
+
+	const embeds = message.embeds;
```

Comment:
Something's not right with indent here ;D

Not sure why it wasn't caught by eslint.